### ISSUE TITLE
Extra form field -  Add new field type is not working 

### ISSUE THREAD

**Issue by mistry-jignesh** [wpfoodmanager/wp-food-manager#126]
**Refernce issue id is** #21 .

 It allow to add field in Ingredients, Neutritions, Units .

Add new field type is not working .

Please check all options and correct it.

![image](https://user-images.githubusercontent.com/75515088/196665340-faac94d6-4498-4c75-b41c-cdd702af01cd.png)

**Comment by kushang-tailor** [comment]
Hello @mistry-jignesh ,

I have fixed this issue and also checked and verified all fields type except Term Checklist, Term Multiselect and Term Select.

If you found any issue regarding this, Please add that issue in next milestone.

**Comment by mistry-jignesh** [comment]
Verified this issue. I have open new issues for **extra field type** .

For Term Checklist, Term Multiselect and Term Select . I have open issue #145.

https://github.com/wpfoodmanager/wp-food-manager/issues/145


As per @kushang-tailor  - **It is not possible to add field in Ingredients, Nutritions, Units. Because this all 3 are wordpress taxonomies for the Custom post type Food manager.**


Here I closed the issue.
